src/server/frontend_xwayland/xwayland_log.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2018 Marius Gripsgard <marius@ubports.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 or 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MIR_FRONTEND_XWAYLAND_LOG_H
+#define MIR_FRONTEND_XWAYLAND_LOG_H
+
+#include <mir/log.h>
+#include <stdlib.h>
+
+namespace mir
+{
+inline void log_verbose(std::string const& message)
+{
+    if (getenv("MIR_X11_VERBOSE_LOG"))
```

Comment:
Is this intended as a temporary measure? Or part of the proposed implementation?

It doesn't match the control of logging elsewhere in the Mir codebase.

Reply:
This was just something make quickly to make it more silent (it will dump a lot to stdout). Not sure how much need it will be to have it this verbose later on though.